libc/src/stdlib/CMakeLists.txt

```diff
@@ -52,6 +52,28 @@ add_entrypoint_object(
     libc.config.linux.app_h
 )
 
+add_entrypoint_object(
+  strfromf
+  SRCS
+    strfromf.cpp
+  HDRS
+    strfromf.h
+  DEPENDS
+    .str_from_util
+)
+
+add_header_library(
+  str_from_util
+  HDRS
+    str_from_util.h
+  DEPENDS
+    libc.src.stdio.printf_core.converter_atlas
```

Comment:
this should be `libc.src.stdio.printf_core.converter`, `converter_atlas` doesn't have its own target.

Reply:
Ok, understood.